src/EventStore.ClientAPI/EventStoreConnection.cs

```diff
@@ -193,9 +193,21 @@ public static IEventStoreConnection
 			Ensure.NotNull(connectionSettings, "connectionSettings");
 			Ensure.NotNull(clusterSettings, "clusterSettings");
 
-			var discoverClient = new HttpAsyncClient(
-				clusterSettings.GossipTimeout,
-				connectionSettings.CustomHttpMessageHandler);
+			var handler = connectionSettings.CustomHttpMessageHandler;
+			if (handler is null) {
+				if (!connectionSettings.ValidateServer) {
+#if NET452
+					connectionSettings.Log.Info(
```

Comment:
I was going to suggest that this log message could be converted to a warning, but it looks like the `ILogger` interface does not have `Warn` method.